### ISSUE TITLE
Parameterise postgres events table name.

### ISSUE THREAD

**Issue by johncowie** [logicblocks/event.store#1]
It'd be nice to add the option for users of the postgres adapter to be able to specify the name of the table that they'd like the events to be stored in, for instance in case they already have an `events` table that they are using for something else.

**Comment by tobyclemson** [comment]
Agree, it's on the "To Do" list in [NOTES.md](https://github.com/logicblocks/event.store/blob/main/NOTES.md), maybe those should be turned into issues instead.

**Comment by tobyclemson** [comment]
This is now supported. Still need to update documentation so will leave the issue open for now.

**Comment by tobyclemson** [comment]
I'd like to parameterise more than just the table name and instead allow users to provide some kind of schema which the adapter uses to map event data into the database, so have renamed this issue to encompass that too.
